### PR TITLE
Minor Twist to clarify the attendance count

### PR DESCRIPTION
**Pull request by cathyhongzhang**
Add this clarification per last meeting discussion

### CODE REVIEW

**Comment by nerdyyatrice** [comment]
LGTM

File: GOVERNANCE.md [hunk 1]
```diff
@@ -68,6 +68,8 @@ If a vote is taken during a meeting, the follow rules will be followed:
 * A participating company, or nonaffiliated individual, attains voting rights
   by having any of their assigned representative(s) attend 3 out of the last
   4 meetings. They obtain voting rights after the 3rd meeting, not during.
+  If the primary representative can not attend a meeting, his/her
+  delegate's attendance is counted.
```
Comment (duglin) [inline GOVERNANCE.md]: Cathy - I agree with the idea but its kind of awkward to have this sentence separate from the one on line 68.
Would you be ok with this instead:
A participating company, or non-affiliated individual, attains voting rights
by having their assigned representative(s), in aggregate, attend 3 out of
the last 4 meetings.

File: GOVERNANCE.md [hunk 2]
```diff
@@ -67,7 +67,9 @@ If a vote is taken during a meeting, the follow rules will be followed:
 * Each participating company can assign a primary and secondary representative.
 * A participating company, or nonaffiliated individual, attains voting rights
   by having any of their assigned representative(s) attend 3 out of the last
-  4 meetings. They obtain voting rights after the 3rd meeting, not during.
+  4 meetings. If the primary representative can not attend a meeting, his/her
+  delegate's attendance is counted. They obtain voting rights
```
Comment (n3wscott) [inline GOVERNANCE.md]: please use 80 width columns.

**Comment by duglin** [comment]
All - as @cathyhongzhang and I were making the edits based on this week's call, we realized that we were each looking at the edits as solving different issues:
1 - make it clear that either the primary or secondary person's presence at a call would count for that week's attendance 
2 - companies can change who their representatives as needed, in particular if both primary and secondary are not available

Both of these issues should be cleared up but the previous proposal wasn't clear as to which is was addressing. We worked on some new text to address both - I don't think it changes anything about our current process, it just documents it. While we can't totally avoid game playing, to avoid a company sending different people to the calls each week **just** to keep voting rights, we limit the rotation of people down to once a month - which isn't an issue since most companies have rarely changed this info in the past.

While I don't think the new edits are controversial, I didn't want to merge the edits w/o the group getting a chance to look them over since any edits to a governance type of doc should be well vetted. I've added this item to the agenda for next week's call - please review and speak-up prior to the call if you have any concerns or suggested edits.

**Pull request closed without merging**